napari/util/theme.py

```diff
@@ -5,6 +5,7 @@
     'dark': {
         'folder': 'dark',
         'background': 'rgb(38, 41, 48)',
+        'background_dark2': 'rgb(33, 36, 42)',
```

Comment:
not sure I love the name `background_dark2`, maybe `darker_background` or something else is preferable

Reply:
yeah, I don't love it either.  `dark2` there means "2 percent darker" and I considered adding different perecentages, but leaving at just `dark2` made it look like it was just "option #2".  until we have additional ones, I can just make it `darker_background` with a comment that it is 2% so we don't forget

Reply:
hmm yeah - i think `background_darker` + comment is a fine choice for now, and we might revisit this stuff when we do a global pass over the styling sometime in the future with a designer's help